src/Interface/Application/ModuleProxyWidget.cc

```diff
@@ -581,6 +586,26 @@ QVariant ModuleProxyWidget::itemChange(GraphicsItemChange change, const QVariant
   return ret;
 }
 
+//TODO
+LoopDiamondPolygon::LoopDiamondPolygon(QGraphicsItem* parent) : QGraphicsPolygonItem(parent)
+{
+  shape_ << QPointF(-100, 0) << QPointF(0, 70)
+                          << QPointF(100, 0) << QPointF(0, -70)
+                          << QPointF(-100, 0);
+  setPolygon(shape_);
```

Comment:
What are these numbers? Could they be made into const variables and belong privately to the class?

Reply:
More copied code that needs sprucing up--it's not enabled yet so I'll clean it up later.